recipe_scrapers/aldisued.py

```diff
@@ -0,0 +1,17 @@
+from ._abstract import AbstractScraper
+
+
+class AldiSued(AbstractScraper):
+    @classmethod
+    def host(cls, domain="aldi-sued.de"):
+        return domain
+
+    def instructions(self):
+        instruction_elements = self.schema.data.get("recipeInstructions", [])
+        return "\n".join(
+            [
+                element.get("text").replace("\xad", "")
```

Comment:
Note: there's a `_utils.normalize_string` function that we could import and extend to do cleanup such as this; I didn't suggest it for use in #1352 because it seemed to conflict with some of the newline handling there, but it could be an option here.

Reply:
I can extend the normalize_string function in a different PR. But I have to adjust 10 affected testcases as well. OK?

Reply:
Hm, gotcha.  Does using `normalize_string` improve the quality of the output, reduce the quality, or... is it ambiguous/hard-to-tell?

Reply:
> is it ambiguous/hard-to-tell?

I take this part of your statement :-)

Reply:
Ok; I don't have much of an opinion about it -- it seems to be largely single-space after full-stop, or double-space after full-stop?   So.. you can decide either way, I think.  I'll keep thinking about it in case I later think of some reason we should prefer one or the other generally, but I doubt it.